src/mesh_element/Element_Definition.cc

```diff
@@ -77,6 +79,7 @@ Element_Definition::Element_Definition(Element_Type const &type_)
   case POLYHEDRON:
     dimension = 3;
     break;
+
   case POLYGON:
     dimension = 2;
     break;
```

Comment:
I know you didn't edit this code, but it looks like `case POLYGON` is not covered by unit tests.  Is this something that you could easily add in this PR?